src/Player.Vm.Api/Infrastructure/Extensions/TelemetryExtensions.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright 2022 Carnegie Mellon University. All Rights Reserved.
+// Released under a MIT (SEI)-style license. See LICENSE.md in the project root for license information.
+
+using Microsoft.Extensions.DependencyInjection;
+using Player.Vm.Api.Infrastructure.Options;
+using System;
+using System.Collections.Generic;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+
+namespace Player.Vm.Api.Infrastructure.Extensions
+{
+    public static class TelemetryExtensions
+    {
+        public static void AddTelemetry(this IServiceCollection services, AuthorizationOptions authOptions)
```

Comment:
Is this not used?

Reply:
It is not.  It was in one iteration, but not any more.  I did remove it from the Player API.  Thanks for catching that.